clientconn.go

```diff
@@ -1610,7 +1659,12 @@ func parseTarget(target string) (resolver.Target, error) {
 // - user specified authority override using `WithAuthority` dial option
 // - creds' notion of server name for the authentication handshake
 // - endpoint from dial target of the form "scheme://[authority]/endpoint"
-func determineAuthority(endpoint, target string, dopts dialOptions) (string, error) {
+//
+// Stores the determined authority in `cc.authority`.
```

Comment:
Optional nit: specify in what conditions an error is returned. (i.e. authority from transport creds != authority specified in Dial Option)

Reply:
Done.